### PR TITLE
X980 worknumber for released labware

### PR DESCRIPTION
**Pull request by seenanair**
- Can select a worknumber for each released labware
- 'Select all' work number option
- Worknumber is optional
- Displays the worknumber assigned to the labware as the default value on scan

### CODE REVIEW

File: src/graphql/queries/GetSuggestedWorkForLabware.graphql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+query GetSuggestedWorkForLabware($barcodes: [String!]!) {
+  suggestedWorkForLabware(barcodes: $barcodes) {
+    suggestedWorks {
+      ...SuggestedWorkFields
+    }
+    works {
```
Comment (khelwood) [inline src/graphql/queries/GetSuggestedWorkForLabware.graphql:6]: I don't think you need to get the `works` stuff here at all: as far as I can tell you only use the `suggestedWorks`.

Comment (seenanair) [reply src/graphql/queries/GetSuggestedWorkForLabware.graphql]: Updated. Thank you

File: src/mocks/handlers/labwareHandlers.ts [hunk 2]
```diff
@@ -93,6 +96,24 @@ const labwareHandlers = [
...
+    (req, res, ctx) => {
+      const work = workFactory.build({ workNumber: 'SGP1008' });
+      return res(
+        ctx.data({
+          suggestedWorkForLabware: {
+            suggestedWorks: [...req.variables.barcodes].map((barcode) => ({
+              workNumber: work.workNumber,
+              barcode: barcode
+            })),
+            works: [work]
```
Comment (khelwood) [inline src/mocks/handlers/labwareHandlers.ts:112]: I think `works:` isn't needed, in line with the last change

Comment (seenanair) [reply src/mocks/handlers/labwareHandlers.ts]: Yes, updated

**Review by khelwood — APPROVED** [review]

**Pull request merged**